### ISSUE TITLE
Bug Report: Boyfriend's Vocals in Dadbattle Erect and Thorns Erect are Muffled and Quiet Compared to OST Versions 

### ISSUE THREAD

**Issue by JackXson-Real** [FunkinCrew/Funkin#2874]
### Issue Checklist

- [x] I have read the [Contributing Guide](https://github.com/FunkinCrew/Funkin/blob/main/docs/CONTRIBUTING.md)
- [x] I have checked the Issues/Discussions pages to see if my issue has already been reported
- [x] I have properly titled my issue

### Platform

Compiled from GitHub Source Code

### Browser

None

### Version

0.5.3 Develop Branch

### Description (include any images, videos, errors, or crash logs)

Basically what title says, this has been an issue since 0.3.0 and this needs to be fixed 😭 boyfriend sounds like he underwater

### Steps to Reproduce

1. Play Dadbattle Erect or Thorns Erect
2. Enjoy boyfriend's underwater vocals

![Image](https://github.com/user-attachments/assets/1c39dabd-0344-4d68-a6bf-4f0095825414)

**Comment by JVNpixels** [comment]
same with dadbattle erect, bfs vocals feel a bit muffled.

**Comment by AbnormalPoof** [comment]
I feel like this is less of an issue and more of a creative decision by the composer. Updating existing songs isn't easy, since there's the OST on streaming platforms to account for, and that ship has already sailed.

**Comment by JackXson-Real** [comment]
Update: This issue has been rewritten since I don't like how it was before, and it was kind of vague.
